apps/web/components/Chat.tsx

```diff
@@ -1,19 +1,18 @@
-import { useEffect, useRef, useState } from "react"
-import { Button } from "./Button"
-import { type ChatGPTMessage, ChatLine, LoadingChatLine } from "./ChatLine"
-import { useCookies } from "react-cookie"
-import { DISCORD_URL, DOWNLOAD_URL } from "./common"
-import * as ai from "window-ai"
+import { useEffect, useRef, useState } from "react";
+import { Button } from "./Button";
+import { type ChatGPTMessage, ChatLine, LoadingChatLine } from "./ChatLine";
+import { useCookies } from "react-cookie";
+import { DISCORD_URL, DOWNLOAD_URL } from "./common";
 
-const COOKIE_NAME = "nextjs-example-ai-chat-gpt3"
+const COOKIE_NAME = "nextjs-example-ai-chat-gpt3";
```

Comment:
should we standardize to no semicolons and put eslint config in a package?

Reply:
yep just haven't gotten around this yet!

Reply:
adding prettier and eslint to the root